src/main/io/osd.c

```diff
@@ -352,6 +354,39 @@ static void osdFormatVelocityStr(char* buff, int32_t vel)
     }
 }
 
+/**
+ * Converts wind speed into a string based on the current unit system, using
+ * always 3 digits and an additional character for the unit at the right. buff
+ * is null terminated.
+ * @param ws Raw wind speed in cm/s
+ */
+#ifdef USE_WIND_ESTIMATOR
+static void osdFormatWindSpeedStr(char *buff, int32_t ws, bool isValid)
+{
+    int32_t centivalue;
+    char suffix;
+    switch (osdConfig()->units) {
+        case OSD_UNIT_UK:
+            FALLTHROUGH;
+        case OSD_UNIT_IMPERIAL:
+            centivalue = (ws * 224) / 100;
+            suffix = SYM_MPH;
+            break;
+        case OSD_UNIT_METRIC:
+            centivalue = (ws * 36) / 10;;
```

Comment:
There are 2 semicolons at the end of this line

Reply:
Nice catch! Fixed :-)